### ISSUE TITLE
Support for old Raspberry pi 1 (armv6)

### ISSUE THREAD

**Issue by Jobba7** [itzg/docker-minecraft-server#2055]
### Enhancement Type

Not sure

### Describe the enhancement

I want to run a minecraft vanilla server on an old Raspberry pi 1 (armv6) (Raspberry Pi Model B Rev 2). Is this possible?

**Comment by itzg** [comment]
This is already possible:

https://github.com/itzg/docker-minecraft-server#running-on-raspberrypi

...however, set your expectations low. It will be incredibly slow and will probably not have enough memory. 

Disabling the healthcheck at least on first run will probably be a good idea

https://github.com/itzg/docker-minecraft-server#healthcheck

**Comment by Jobba7** [comment]
Yes but, if i run `docker run -d -it -p 25565:25565 --name mc -e EULA=TRUE -e MEMORY=400m itzg/minecraft-server:latest` then the output is `docker: no matching manifest for linux/arm/v6 in the manifest list entries.`

And if i run `docker run -d -it -p 25565:25565 --name mc -e EULA=TRUE -e MEMORY=400m --platform linux/arm/v7 itzg/minecraft-server:latest` then the container stops instantly after starting with this error in the log: `No log line matching the '' filter`

Is there a build supported for `linux/arm/v6`?

**Comment by itzg** [comment]
I'm sorry, I didn't notice the exact model you described. I could probably add that to the build since required binaries include armv6. Honestly, it's not really worth the effort. If you're wanting to run anything newer than 1.12.x then it will be incredibly slow, as in 60 minutes for just startup.

**Comment by Jobba7** [comment]
I want to give it a try, just for answering the question: can it run minecraft? If it's not too much effort I would be happy :)

**Comment by itzg** [comment]
I'll queue this up to take a quick stab at it.

**Comment by camalot** [comment]
everything I see says that Java 11 and later isn't supported on ARMv6? so it would be just java8 that would have support for ARMv6, correct?

**Comment by itzg** [comment]
Good point @camalot , I didn't think about that aspect.
